requirements-docs.in

```diff
@@ -37,7 +37,7 @@ pem
 pyjks >= 19 # pyjks < 19 depends on pycryptodome, which conflicts with dyn's usage of pycrypto
 pyjwt
 pyOpenSSL
-redis
+redis==4.0.2  # prevent conflict between celery and fakeredis
```

Comment:
Is there an issue we can refer to, so we can unpin this in the future, once the issue has been addressed?

Reply:
There is a [pending PR to update fakeredis dependencies](https://github.com/jamesls/fakeredis/pull/324) that should resolve this, added a reference.

Reply:
thank you